integration-tests/src/test/java/oracle/kubernetes/operator/ITServerDiscovery.java

```diff
@@ -408,6 +282,12 @@ public void testOPRestartDeadMS() throws Exception {
     logger.info("SUCCESS - " + testMethodName);
   }
 
+  /**
```

Comment:
javadoc wrong
refer to https://www.oracle.com/technetwork/java/javase/tech/index-137868.html

Reply:
fixed

I also removed javadoc for three private methods because I saw this:

Javadoc must be provided for all public packages, classes, and methods, and must include all parameters and returns. Javadoc is not required for methods that override or implement methods that are already documented.

on https://github.com/oracle/weblogic-kubernetes-operator/blob/master/site/v2.0.1/developer.md#coding-standards

Please let me know if you think I should add them back

Reply:
yes that is totally fine.

Reply:
please push your commit, i cannot see you changes. thanks.

Reply:
was running the test in my local. will push now